enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -92,41 +72,65 @@ scrape_configs:
         regex: container_(network_tcp_usage_total|tasks_state|cpu_usage_seconds_total|memory_failures_total|network_udp_usage_total)
         action: drop
 
-  # Scrape config for service endpoints.
+  # pushgateway
+
+  - job_name: 'pushgateway-exporter'
+    honor_labels: true
+
+    kubernetes_sd_configs:
+      - role: service
+
+    relabel_configs:
+      - source_labels: [__meta_kubernetes_service_annotation_{{ dot2underscore .PrometheusDomain }}_probe]
+        action: keep
+        regex: pushgateway
```

Comment:
Don't we need the `.MonitorTypeRules` here as well? To add workload, etc. to pods that are using pushgateway.

Reply:
there is no change to pushgateway or blackbox exporter.  The metadata would describe the exporter, not the workload pushing data to the exporter.